app/src/main/java/com/keylesspalace/tusky/components/timeline/viewmodel/NetworkTimelineRemoteMediator.kt

```diff
@@ -71,7 +80,13 @@ class NetworkTimelineRemoteMediator(
                 val expanded = oldStatus?.isExpanded ?: activeAccount.alwaysOpenSpoiler
                 val contentCollapsed = oldStatus?.isCollapsed ?: true
 
+                var inReplyToAccount:TimelineAccount? = null
+                if (status.inReplyToAccountId != null) {
+                    inReplyToAccount = accountDao.get(status.inReplyToAccountId)?.toAccount(gson)
```

Comment:
I don't think the account will always exist in the local database.

Consider: You follow user B, they reply to user A, who you do not follow.

This works for a reblog, because the `reblog` field contains the complete status, which includes the full information of the original author (https://docs.joinmastodon.org/entities/Status/#reblog).

A status that's a reply doesn't, it only contains `in_reply_to_account_id` (https://docs.joinmastodon.org/entities/Status/#in_reply_to_account_id).

It *might* also contain a `mentions` object, but AFAIK there's no requirement that a reply also mention the person they're replying to, and even if there is a `mentions` object it doesn't contain the display name of the person that's being replied to.

Reply:
It does work so far (for a week) on my timeline.

Not sure if the api actually provides arbitrary account information but I'm pretty sure we wouldn't want to do an api call for this information.

At the moment if an account is missing the text line would be missing.

(So is there an alternative?)

Reply:
Two strings, "Replied" and "Replied to %s", select the correct one based on whether or not we know the account info.

Reply:
ok